### ISSUE TITLE
Album and artist information gone after updating to ownCloud8 and enabling encryption

### ISSUE THREAD

**Issue by devurandom** [owncloud/music#342]
### Steps to reproduce
1. Open Music app
### Expected behaviour

I should see the artist, album and track names
### Actual behaviour

I see "Unknown artist" / "Unknown album" and raw filenames
### Server configuration

**Operating system**: Gentoo/Linux

**Web server:** Lighttpd 1.4.35

**Database:** MariaDB 10.0.15

**PHP version:** 5.5.21

**ownCloud version:** 8.0.0.7

**Updated from an older ownCloud or fresh install:** Yes

**List of activated apps:**
- Music 0.3.4

**The content of config/config.php:**

```
<?php
$CONFIG = array (
  'instanceid' => 'XXX',
  'passwordsalt' => 'XXX',
  'trusted_domains' => 
  array (
    0 => 'XXX',
  ),
  'datadirectory' => 'XXX',
  'dbtype' => 'mysql',
  'version' => '8.0.0.7',
  'dbname' => 'XXX',
  'dbhost' => 'XXX',
  'dbtableprefix' => 'XXX',
  'dbuser' => 'XXX',
  'dbpassword' => 'XXX',
  'installed' => true,
  'ldapIgnoreNamingRules' => false,
  'forcessl' => true,
  'theme' => '',
  'maintenance' => false,
  'mail_smtpmode' => 'smtp',
  'mail_smtpsecure' => 'tls',
  'mail_smtphost' => 'XXX',
  'mail_smtpport' => 'XXX',
  'mail_domain' => 'XXX',
  'mail_from_address' => 'no-reply',
  'loglevel' => '0',
  'secret' => 'XXX',
  'forceSSLforSubdomains' => true,
);
```

**Are you using external storage, if yes which one:** No

**Are you using encryption:** Yes
### Client configuration

**Browser:** 35.0.1

**Operating system:** Gentoo/Linux
#### Web server error log

```
…
```
#### ownCloud log (data/owncloud.log)

Lots of "Accessing the API class is deprecated! Use the appropriate services instead!" for various parts of ownCloud, and "Ready for a paged search" once for every file retrieved via WebDAV.

```
…
```
#### Browser log

Nothing exceptional:

```
Using //@ to indicate sourceMappingURL pragmas is deprecated. Use //# instead jquery.min.js:1
Using //@ to indicate sourceMappingURL pragmas is deprecated. Use //# instead jquery-migrate.min.js:3
Content Security Policy: The page's settings blocked the loading of a resource at self ("script-src https://… 'unsafe-eval'"). music
window.controllers is deprecated. Do not use it for UA detection. ace.js:1716
Use of getPreventDefault() is deprecated.  Use defaultPrevented instead. jquery.min.js:5
```
### Additional information

After disabling encryption and decrypting all files via the personal settings page, the album, artist and track information reappears.

**Comment by MightyCreak** [comment]
On my side, I just updated to ownCloud 8 and all I have when I open Music is just a blank page...

**Comment by devurandom** [comment]
@MightyCreak You need 7a41981a80c11f4d86609a6e57709aec15231f37 to fix the blank page.

**Comment by MightyCreak** [comment]
@devurandom Yes! That's it! Thanks!

**Comment by MorrisJobke** [comment]
@devurandom The music app doesn't work with encryption really good yet. There are plans to fix this, but that's more a issue of the owncloud core.

**Comment by paulijar** [comment]
I tested encryption with Nextcloud 13.0.0-alpha, Default encryption module, Music app 0.5.4, and Tomahawk 0.8.4 for Windows. All the album and artists names and other metadata were extracted and shown correctly. My best guess is that this has got fixed by the commit 330fc142a27d44. Closing.
